modules/packages/Sort.chpl

```diff
@@ -591,55 +591,65 @@ proc binaryInsertionSort(Data: [?Dom] ?eltType, comparator:?rec=defaultComparato
  */
 proc mergeSort(Data: [?Dom] ?eltType, minlen=16, comparator:?rec=defaultComparator) {
   chpl_check_comparator(comparator, eltType);
-  _MergeSort(Data, minlen, comparator);
+  _MergeSort(Data, Dom.low, Dom.high+1, minlen, comparator);
```

Comment:
I think it'd make more sense to pass Dom.low and Dom.high because Chapel ranges normally include the upper bound.

Reply:
I like the version of this PR that doesn't add 1 to high, since the bounds are inclusive